### DISCUSSION TITLE
is react.py production ready?

### DISCUSSION THREAD

**Discussion by ghost** [idom-team/idom#1166] [Question]
Hi, I would love to ask couple of questions.
1. is react.py production ready?
2. what are limitations & current big blocking issues to use in production (if there is any)?
3. how long would it take to have docs ready too?
Thanks :)

**Comment by rmorshea** [comment]
I think the answer to this likely depends on your use case. For example, authentication is still a bit awkward to implement at the moment, but that may not matter if you handle auth some other way (e.g. API Gateway in AWS) or you use a VPN and can rely on network level security. Can you provide a bit more info on how you plan on using ReactPy?

**Comment by Archmonger** [comment]
1. [ReactPy-Django](https://github.com/reactive-python/reactpy-django) is currently the only production ready form of ReactPy. We are working on getting our other backends to the same level.
2. Embedding JavaScript within ReactPy is still a bit awkward, and we are working on that.
3. We are working on the docs but can't give firm timelines. All our work is done on this project is volunteer hours.

**Reply by rmorshea** [reply]
As a note, the django integration does not suffer from the same auth difficulties that the standalone library does at the moment.

**Reply by Archmonger** [reply]
For jupyter, you will need to use [ReactPy Jupyter](https://github.com/reactive-python/reactpy-jupyter).

**Comment by retsyo** [comment]
Any list of project or company who uses reactpy?

**Comment by kumaraguru1735** [comment]
Reflex has proper docs, and all feature's , even its on beta stage
This project donot have any docs, None of the important features implemented
I will say wait for some year's.. until do some alternative stuff
